notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1549 @@
+{
```

Comment:
<p>Grammar: "Let's test if it runs locally before passing it to Vertex for training."</p><p> </p><p>Also add -&gt; "This will help identify and fix any errors in our code locally"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Done!</p>